system/otherLoadTest/playlist.xml

```diff
@@ -22,6 +22,23 @@
 		<variations>
 			<variation>NoOptions</variation>
 		</variations>
+		<command>$(SYSTEMTEST_CMD_TEMPLATE) -test=MixedLoadTest -test-args=$(Q)timeLimit=5m$(Q) -java-args=$(Q)--add-opens java.base/java.lang=ALL-UNNAMED$(Q);\
```

Comment:
` -java-args` is already set in `$(SYSTEMTEST_CMD_TEMPLATE)`. In the console output, `-java-args` set twice 
- `-java-args=`
- `-java-args="--add-opens java.base/java.lang=ALL-UNNAMED"`

This worked in Grinder because we are running no options. Once we enable other JVM options, I suspect this will break. Unless STF aggregates all `-java-args` values.

```
00:04:10.165  mkdir -p "/home/jenkins/workspace/Grinder/openjdk-tests/TKG/../TKG/test_output_16101271418479/MiniMix_5min_0"; \
00:04:10.166  cd "/home/jenkins/workspace/Grinder/openjdk-tests/TKG/../TKG/test_output_16101271418479/MiniMix_5min_0"; \
00:04:10.166  perl /home/jenkins/workspace/Grinder/openjdk-tests/TKG/../../jvmtest/system/otherLoadTest/..//stf/stf.core/scripts/stf.pl -test-root="/home/jenkins/workspace/Grinder/openjdk-tests/TKG/../../jvmtest/system/otherLoadTest/..//stf;/home/jenkins/workspace/Grinder/openjdk-tests/TKG/../../jvmtest/system/otherLoadTest/..//openjdk-systemtest;/home/jenkins/workspace/Grinder/openjdk-tests/TKG/../../jvmtest/system/otherLoadTest/..//openj9-systemtest" -systemtest-prereqs="/home/jenkins/workspace/Grinder/openjdk-tests/TKG/../../jvmtest/system/otherLoadTest/..//systemtest_prereqs" -java-args=' ' -results-root="/home/jenkins/workspace/Grinder/openjdk-tests/TKG/../TKG/test_output_16101271418479/MiniMix_5min_0" -test=MixedLoadTest -test-args="timeLimit=5m" -java-args="--add-opens java.base/java.lang=ALL-UNNAMED";\

```

Reply:
Great catch @llxia . I've updated the PR to use a different STF option (-java-args-execute-initial) to supply the additional JVM option. 

Test : Grinder/12991 

So, now we have both the options coming in from Mode150 (e.g. `-XX:+UseCompressedOops`), as well as `--add-opens java.base/java.lang=ALL-UNNAMED`

```
14:10:02  STF 11:10:01.596 - Running command: /home/jenkins/workspace/Grinder/openjdkbinary/j2sdk-image/bin/java --add-opens java.base/java.lang=ALL-UNNAMED -XX:+UseCompressedOops
```